frontend/cypress/support/commands.ts

```diff
@@ -114,9 +100,8 @@ const finishLogin = (authEndpoint: string, username: string, password: string, c
       csrf: csrf
     }
   }).then(resp => {
-    const kialiURLWithToken = new URL(resp.redirects[1].replace('302: ', ''));
```

Comment:
I replaced this with .pop(), as the previous version threw some errors on ROSA and this seemed to fix things.